sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/Capabilities.java

```diff
@@ -100,6 +101,11 @@ public boolean supportsV1APIByDefault() {
         return hasOrExceedsVersion(1, 11);
     }
 
+    public boolean supportsRegionAwareness() {
+        // This feature is in BETA for 1.11, so requires explicit opt-in by end-users.
+        return SchedulerConfig.fromEnv().isregionAwarenessEnabled() && hasOrExceedsVersion(1, 11);
```

Comment:
Does it make more sense for this to be a setting on either the `scheduler` or on a per-pod basis in the service spec?

I don't know of any other service behavior that is controlled with envvars other than uninstall, and I'm not a huge fan of having to set a magic ENVVAR as a service developer.

Reply:
My stance is this is ok for beta, but we should determine how granular we want service developers to be able to go, and how we should officially expose it to them.

Reply:
1. I think it makes sense at the scheduler layer.  You're opting into a behavior all up.  It's a feature flag.
1. If you want to use it for one pod and not another, then set your placement constraints appropriately.
1. Regarding the ENVVAR thing, it could be a first class citizen in the YAML (capabilities section), but that's not in scope here.